.gdbinit

```diff
@@ -0,0 +1,13 @@
+add-auto-load-safe-path /home/aaron/Projects/zip-info/.gdbinit
```

Comment:
Is there a way to make this general to any user's home folder on any platform? Or can this line just be removed?